<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferFrame</class>
 <widget class="QFrame" name="TransferFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;
color: rgb(0, 0, 0);
background-color: rgb(250, 250, 250);
font-family: Poppins;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1,1,1,1">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; color: #5f5f5f; </string>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="m_labelEdit">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="placeholderText">
      <string>Enter a label for this address to add it to your address book</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="m_pasteButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/images/paste.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="m_removeButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/images/minus.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; color: #5f5f5f; </string>
     </property>
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="m_amountSpin">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>200000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>558</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="m_addressEdit">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="m_addressBookButton">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/images/contact.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; color: #5f5f5f; </string>
     </property>
     <property name="text">
      <string>Pay to</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; color: #5f5f5f; </string>
     </property>
     <property name="text">
      <string>Crypto comment</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QLineEdit" name="m_transactionCommentEdit">
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 0px; 
background-color: rgb(238, 238, 238);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_addressEdit</tabstop>
  <tabstop>m_labelEdit</tabstop>
  <tabstop>m_amountSpin</tabstop>
  <tabstop>m_addressBookButton</tabstop>
  <tabstop>m_pasteButton</tabstop>
  <tabstop>m_removeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_removeButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>addressBookClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>786</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>TransferFrame</receiver>
   <slot>pasteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>TransferFrame</receiver>
   <slot>addressEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_transactionCommentEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TransferFrame</receiver>
   <slot>commentEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressBookClicked()</slot>
  <slot>pasteClicked()</slot>
  <slot>addressEdited(QString)</slot>
  <slot>commentEdited(QString)</slot>
 </slots>
</ui>
